.circleci/config.yml

```diff
@@ -191,7 +191,7 @@ mandatory_jobs: &mandatory_jobs
   - python 3_7 git 2_7_6
   - pep8 and python 3_8 git 2_25_0
   - coverage and python 3_9 git 2_30_1
-  - sphinx docs and python 3_10 git 2_33_1
+  - docs and python 3_10 git 2_33_1
```

Comment:
FYI, I've just changed the required checks for PR to be mergable to `develop`/`master` to accommodate for the new name